### DISCUSSION TITLE
Automatic creation of albums based on the folders in the library

### DISCUSSION THREAD

**Discussion by emanueletocci** [immich-app/immich#8306] [General]
Hi, I'm finally using immich with my nas storage as external library. Is possible to create **automatically** some albums based on the folders in the library? My nas storage look like this: //Archivio/%year%/%event%/photo_name .
Are there some ways to create album based on this path? For example i would like to have %event% (or %year) based album in my immich istance... is it already possible?

**Comment by mmomjian** [comment]
Duplicate of #4279
